### ISSUE TITLE
Colour variables references

### ISSUE THREAD

**Issue by pascalduez** [SassDoc/sassdoc-theme-default#78]
A colour variable referencing another colour variable is not getting the rendered preview.

```scss
/// Links color
/// @type Color
$link-color: $primary-color !default;
``` 

It's been already evoked in [this comment](https://github.com/SassDoc/sassdoc-theme-default/issues/28#issuecomment-76375299).

@FWeinb suggestion:

> This should in fact be possible. We might need to add some helper methods to SassDoc-extras to generate a variable-value map and than use that to get the color. This implies that the used variable is in fact documented using SassDoc.

**Comment by HugoGiraudel** [comment]
Yup.

**Comment by pascalduez** [comment]
I started working on this, will try to open the PR soon.
It's a new filter in `sassdoc-extras`.

We might want to introduce a new key on the data items. Something like `item.colorPreview`. 
Otherwise if we replace the value (a variable) with the color, the code preview will display the replaced value and not the original one.

**Comment by HugoGiraudel** [comment]
Makes sense. Ping @valeriangalliat @FWeinb.

**Comment by FWeinb** [comment]
Let's be more generic with the name and call it `resolvedVariable`. That why we can extend it if we ever need the values of other variable types.

**Comment by valeriangalliat** [comment]
+1 for `resolvedVariable`!

**Comment by pascalduez** [comment]
`resolvedValue` sounds good.

Thanks for the feedback guys!

**Comment by pascalduez** [comment]
Available as of `sassdoc-extras@2.4.0` and `sassdoc-theme-default@2.4.0`.
